### ISSUE TITLE
AS 3.7.1: Problem running ECMWF jobs

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#181]
In GitLab by @mcastril on Aug 30, 2016, 17:58

Leaving aside some problems that have been ocurring recently with ECMWF connection (and ECACCESS), I have been observing a problem when running jobs in this HPC using Autosubmit 3.7.1. 

My scenario is: Some experiments from the testing suite, created using Autosubmit 3.6.0. When I load Autosubmit 3.7.1, the REMOTE_SETUP does nothing. It creates the REMOTE_SETUP_STAT file, but looking at the logs, just after the line "set -xuve" is not printing anything.

However, if I take the REMOTE_SETUP.cmd file and do qsub REMOTE_SETUP.cmd it seems to run ok, and also creating again the experiment, using Autosubmit 3.6.0, the REMOTE SETUP works. 

The only differences between REMOTE_SETUP.cmd outputted by 3.7.1 and 3.6.0 are the .out and .err path setting, and one extra  #!/bin/bash

I will perform more tests, but currently there is a returning issue when connecting to ECMWF, that is making impossible even to do these tests.

**3.7.1**

```
#!/usr/bin/env bash

###############################################################################
#                   REMOTE_SETUP t00u EXPERIMENT
###############################################################################
#
#PBS -N t00u_REMOTE_SETUP
#PBS -o /scratch/ms/spesiccf/c3e/t00u/LOG_t00u/t00u_REMOTE_SETUP.20160830163936.err
#PBS -e /scratch/ms/spesiccf/c3e/t00u/LOG_t00u/t00u_REMOTE_SETUP.20160830163936.out
#PBS -q ns
#PBS -l walltime=02:00:00
#PBS -l EC_billing_account=spesiccf
#
###############################################################################

###################
# Autosubmit header
###################

job_name_ptrn='/scratch/ms/spesiccf/c3e/t00u/LOG_t00u/t00u_REMOTE_SETUP'
echo $(date +%s) > ${job_name_ptrn}_STAT
```

**3.6.0**

```
#!/bin/bash
###############################################################################
#                   REMOTE_SETUP t00u EXPERIMENT
###############################################################################
#
#PBS -N t00u_REMOTE_SETUP
#PBS -q ns
#PBS -l walltime=02:00:00
#PBS -l EC_billing_account=spesiccf
#
###############################################################################

#!/usr/bin/env bash
###################
# Autosubmit header
###################

job_name_ptrn='/scratch/ms/spesiccf/c3e/t00u/LOG_t00u/t00u_REMOTE_SETUP'
echo $(date +%s) > ${job_name_ptrn}_STAT

```

@dmanubens @ jlope2

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Sep 6, 2016, 13:30

Done! Solved in new version

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Sep 9, 2016, 17:20

Status changed to closed by commit e40eac8ec170433ee1557db76ece46c97896905f
